src/components/design_system/input/Input.tsx

```diff
@@ -1 +1,30 @@
-export {};
+import React from "react";
+import TextField, { TextFieldProps } from "@mui/material/TextField";
+import { styled } from "@mui/material/styles";
+
+export const TextInput = styled(TextField)<TextFieldProps>(({ theme }) => ({
```

Comment:
can either use `theme` colors from `theme.ts` instead of `var(...)` below or remove it altogether